agent/engine/docker_image_manager.go

```diff
@@ -476,11 +489,17 @@ func (imageManager *dockerImageManager) getNonECSImages(ctx context.Context) []I
 			seelog.Errorf("Error inspecting non-ECS image: (ImageID: %s), %s", imageID, err)
 			continue
 		}
+		createTime := time.Time{}
+		createTime, err = time.Parse(time.RFC3339, resp.Created)
+		if err != nil {
+			seelog.Errorf("Error parse the inspected non-ECS image created time (ImageID: %s), %s", imageID, err)
```

Comment:
This log level can just be WARN, considering this will not impact real functionality and we don't want to make customer panic when they see this as an error.